src/Utils/NormalizeZlibDeflateTransformStream.ts

```diff
@@ -0,0 +1,23 @@
+import { Transform, TransformCallback } from 'stream';
+
+export class NormalizeZlibDeflateTransformStream extends Transform {
+  private hasCheckedHead = false;
+  // eslint-disable-next-line @typescript-eslint/naming-convention
+  public _transform(
+    chunk: any,
+    encoding: BufferEncoding,
+    callback: TransformCallback
+  ) {
+    if (!this.hasCheckedHead && chunk.length !== 0) {
+      // ADHOC: detects raw deflate: https://stackoverflow.com/a/37528114
+      if (chunk[0] !== 0x78) {
```

Comment:
Use the `compare` static method instead
```suggestion
      if (chunk.compare(this.header, 0, 1, 0, 1) === 0) {
```